test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests.csproj

```diff
@@ -14,6 +14,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <DotNetCliToolReference Include="dotnet-xunit" Version="$(DotNetXUnitCliVer)" />
+    <PackageReference Include="Wiremock.Net" Version="1.5.18" />
```

Comment:
Please add version to props file, as for previous line.
Why so old? There is 1.5.20.

Reply:
Seems like they did a few patches since I wrote this code.
Added to the props file now.